environs/bootstrap/config.go

```diff
@@ -111,6 +116,10 @@ var BootstrapConfigSchema = environschema.Fields{
 			CACertKey),
 		Type: environschema.Tstring,
 	},
+	SSHServerHostKeyKey: {
+		Description: "Sets the bootstrapped controllers SSH server host key",
```

Comment:
missing apostrophe

Reply:
Funnily I left this because the CA keys had this issue too and I wanted it consistent, I'll change them too whilst I'm here